src/modules/pages/itemPage/ItemPage.tsx

```diff
@@ -0,0 +1,63 @@
+import { useNavigate, useParams } from 'react-router-dom';
+
+import { AccountType } from '@graasp/sdk';
+import { ItemLoginWrapper } from '@graasp/ui';
+
+import { HOME_PATH } from '@/config/paths';
+import { hooks, mutations } from '@/config/queryClient';
+import {
+  ITEM_LOGIN_PASSWORD_INPUT_ID,
+  ITEM_LOGIN_SIGN_IN_BUTTON_ID,
+  ITEM_LOGIN_USERNAME_INPUT_ID,
+} from '@/config/selectors';
+import PlayerCookiesBanner from '@/modules/cookies/PlayerCookiesBanner';
+
+import ItemForbiddenScreen from '../../item/ItemForbiddenScreen';
+import MainScreen from '../../item/MainScreen';
+import { EnrollContent } from './EnrollContent';
+import { RequestAccessContent } from './RequestAccessContent';
+
+const { useItem, useItemLoginSchemaType, useCurrentMember } = hooks;
+
+const ItemPage = (): JSX.Element | null => {
+  const { itemId } = useParams();
+  const navigate = useNavigate();
+  const { mutate: itemLoginSignIn } = mutations.usePostItemLogin();
+  const { data: item, isLoading: isItemLoading } = useItem(itemId);
+  const { data: itemLoginSchemaType, isLoading: isLoadingItemLoginSchemaType } =
+    useItemLoginSchemaType({ itemId });
+  const { data: currentAccount, isLoading: isLoadingMember } =
+    useCurrentMember();
+
+  if (!itemId) {
+    navigate(HOME_PATH);
+    return null;
+  }
```

Comment:
Should we also consider a null item?

Reply:
A null item might be because you don't have access or you need to item login, it's dealt later in the code.
Here we redirect if the path does not have an id, mostly.